cmd_gen_finetuning.ipynb

```diff
@@ -205,6 +229,40 @@
     "print(get_formatting_func_from_dataset(train_dataset, tokenizer)(eval_dataset[0]))"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from unsloth.chat_templates import get_chat_template\n",
```

Comment:
Please either:
- Remove this code cell and include a link to the [Unsloth documentation](https://docs.unsloth.ai/basics/chat-templates) in the written instructions above it. This code should be optional and removing it will reduce the verbosity of the notebook.
- Merge this code cell with the one above it and below it, and update the written instructions above to explain what the code is doing.